brewchat/src/main/java/com/example/brewchat/ChatService.java

```diff
@@ -96,7 +101,20 @@ public void login(String username, String password) {
         QBAuth.createSession(user, new QBEntityCallbackImpl<QBSession>() {
             @Override
             public void onSuccess(QBSession session, Bundle params) {
-                EventBus.getDefault().post(new UserLoggedEvent());
+                user.setId(session.getUserId());
+                QBChatService.getInstance().login(user, new QBEntityCallbackImpl() {
+                    @Override
+                    public void onSuccess() {
+                        QBChatService.getInstance().getPrivateChatManager()
+                                .addPrivateChatManagerListener(ChatService.this);
+                        EventBus.getDefault().post(new UserLoggedEvent());
+                    }
+
+                    @Override
+                    public void onError(List errors) {
+                        EventBus.getDefault().post(new AuthenticationErrorEvent(errors));
+                    }
+                });
```

Comment:
Was getting a nullpointer when trying to do anything with the contacts. Needed to make sure we were logged into the ChatService and not just the general QbickBlox session.